geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CreateDiskStoreCommand.java

```diff
@@ -122,7 +126,11 @@ public ResultModel createDiskStore(
     Set<DistributedMember> targetMembers = findMembers(groups, null);
 
     if (targetMembers.isEmpty()) {
-      return ResultModel.createError(CliStrings.NO_MEMBERS_FOUND_MESSAGE);
+      if (!stageDiskStoreConfig) {
```

Comment:
If this "allowing configuration to be created before any server is started" feature is needed, we should be bold about changing this old behavior of returning an error when no server found. No matter this flag is there or not, we should just go ahead and create the configuration in cc.